docs/BestPractices.md

```diff
@@ -39,6 +39,29 @@ FROM node:7.3.0-alpine
 RUN addgroup -S node && adduser -S -g node node 
 ```
 
+Note that the `node` user is neither a build-time nor a run-time dependency and
+it can be removed or altered, as long as the functionality of the application
+you want to add to the container dose not depend on it.
+
+If you do not want nor need the user created in this image you can remove it with
+the following:
+
+```Dockerfile
+RUN userdel -r node
+```
+
+If you need to change the uid/gid of the user you can use:
+
+```Dockerfile
+RUN groupmod -g 999 node && usermod -u 999 -g 999 node
+```
+
+If you need another name for the user execute:
+
+```Dockerfile
+RUN usermod -d /home/newname -l newname node
```

Comment:
I suggest using `myapp` instead of `newname` as that may be less confusing. Also referencing the example name or ID in the above paragraph may also be helpful:

> If you need another name for the user (ex. `myapp`) execute: